django_project/lesson/forms/specification.py

```diff
@@ -0,0 +1,53 @@
+# coding=utf-8
+"""Section forms."""
+
+from django.utils.translation import ugettext_lazy as _
+
+from crispy_forms.helper import FormHelper
+from crispy_forms.layout import (
+    Layout,
+    Fieldset,
+    Field,
+    Submit,
+)
+
+from modeltranslation.forms import TranslationModelForm
+from lesson.models.specification import Specification
+
+
+class SpecificationForm(TranslationModelForm):
+    """Form for creating specification"""
```

Comment:
A specification is a simple requirement that will be included in a lesson.

A specification order is significant - you can sort them so that they are sequenced correctly. The specifications for a lesson will be rendered as a table in the lesson in the order specified.  Here is a simple example of a specification list:

Requirement.       Value
Line colour           Red
Line width             12pt
Line style.             dashed

Each specification can have a note that will be shown as a tooltip in the rendered worksheet, or in the future could be shown underneath the table entries as footnote.